### PR TITLE
Reverse reverted commit

### PR DESCRIPTION
**Pull request by iamchenjiajun**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 Display | Name | Github Profile | Portfolio 
 --------|:----:|:--------------:|:---------:
-![](https://via.placeholder.com/100.png?text=Photo) | GuoAi | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Zeyu Cao | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![MuhammadPhoto](./displayPictures/MuhammadPhoto.jpg) | Muhammad Hozefa | [Github](https://github.com/MuhammadHoze) | [Portfolio](https://github.com/AY2021S1-CS2113-T14-3/tp/blob/master/docs/team/muhammadhozefa.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Chen Jia Jun | [Github](https://github.com/iamchenjiajun) | [Portfolio](docs/team/iamchenjiajun.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Antonio | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | GuoAi | [Github](https://github.com/) | [Portfolio](./team/template.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Cao Zeyu | [Github](hhttps://github.com/Cao-Zeyu) | [Portfolio](./team/Cao-Zeyu.md)
```
Comment (MuhammadHoze) [inline docs/AboutUs.md]: I think @Cao-Zeyu  forgot to remove the extra 'h' from `https://`   Could you help omit the extra 'h' thanks

Comment (iamchenjiajun) [reply docs/AboutUs.md]: Alright, updated 👍

File: docs/UserGuide.md [hunk 2]
```diff
@@ -72,6 +72,8 @@ List of `<optional arguments>`:
 🚩: Duplicate entry of tasks are **allowed**. 
 
 
+🚩: Duplicate task descriptions are allowed. 
+
```
Comment (MuhammadHoze) [inline docs/UserGuide.md]: This line not required anymore

image

File: docs/UserGuide.md [hunk 3]
```diff
@@ -112,6 +114,8 @@ List of `<compulsory arguments>`:
 🚩: Duplicate entry of recurring tasks are **allowed**. 
 
 
+🚩: Duplicate task descriptions are **allowed**. 
+
```
Comment (MuhammadHoze) [inline docs/UserGuide.md]: Same for this, not required anymore

File: src/main/java/seedu/duke/model/item/Link.java [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 package seedu.duke.model.item;
 
+import seedu.duke.DukeException;
+
```
Comment (MuhammadHoze) [inline src/main/java/seedu/duke/model/item/Link.java:6]: can omit out the whitespace

**Comment by MuhammadHoze** [comment]
Alright besides that, all LGTM!

File: src/main/java/seedu/duke/model/item/Link.java (hunk 4 above)
Comment (iamchenjiajun) [reply src/main/java/seedu/duke/model/item/Link.java:6]: This is fine, IntelliJ splits up normal imports from other classes, exception imports and Java classes imports.

Example:
image

**Review by MuhammadHoze — APPROVED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (iamchenjiajun) [reply docs/UserGuide.md]: Ohh I took this change when there was a merge conflict haha.

File: docs/UserGuide.md (hunk 2 above)
Comment (iamchenjiajun) [reply docs/UserGuide.md]: Fixed

File: src/main/java/seedu/duke/model/item/Link.java (hunk 4 above)
Comment (MuhammadHoze) [reply src/main/java/seedu/duke/model/item/Link.java:6]: Ahh alright then

**Comment by iamchenjiajun** [comment]
Merge when the CI is done.

File: docs/UserGuide.md (hunk 3 above)
Comment (MuhammadHoze) [reply docs/UserGuide.md]: ohhh i see, i rephrased it to line 114 instead

**Pull request merged**